src/cart/crt_context.c

```diff
@@ -1910,3 +1952,176 @@ crt_req_force_completion(struct crt_rpc_priv *rpc_priv)
 	crt_req_timeout_track(rpc_priv);
 	D_MUTEX_UNLOCK(&crt_ctx->cc_mutex);
 }
+
+int
+crt_context_quotas_init(crt_context_t crt_ctx)
+{
```

Comment:
if you keep those init/finalize routines internal, you can just return void for both.

Reply:
sure, i can also just inline init during the context init itself if we decide we dont need ability to suspend/resume quotas at runtime.